packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -172,6 +223,14 @@ MeasureAreaToolModel
     return new MeasureAreaToolModel();
   }
 
+  public override decorate(context: DecorateContext): void {
+    super.decorate(context);
+
+    if (this._enableSheetMeasurements && this._currentMousePoint !== undefined && this.toolModel.drawingOrigin !== undefined && this.toolModel.drawingExtents !== undefined && !SheetMeasurementsHelper.checkIfInDrawing(this._currentMousePoint, this.toolModel.drawingOrigin, this.toolModel.drawingExtents)) {
```

Comment:
Feels like the drawing model boundary should be displayed regardless of where the mouse is. It would greatly simplify the 'if' logic.
Also, the `this._currentMousePoint` won't work for mobile devices.